airbyte-webapp/src/components/ui/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "scss/colors";
+
+.container {
+  font-weight: normal;
+  cursor: default;
+}
+
+.item {
+  display: inline-block;
+  cursor: pointer;
+  color: colors.$blue-400;
+  text-decoration: none;
```

Comment:
looks like the `<Link />` component already has alot of these rules, as well as hover. Is there a reason why these are repeated here?

Reply:
The styles are just copy & pasted from the styled components, except for `text-decoration: none;`. I'll have another look though and get rid of any duplicate styles.

Reply:
Updated ✔️